src/v2/cookbook/serverless-blog.md

```diff
@@ -105,7 +106,6 @@ Then create `components/BlogHome.vue` which will be your blog homepage that list
           page: 1,
           page_size: 10
         }).then((res) => {
-          // console.log(res.data)
```

Comment:
Not necessary because it has a real job to do below, so removed.

Reply:
Good idea.